io.neovim.nvim.yaml

```diff
@@ -241,22 +241,24 @@ modules:
       # Install/fix AppData
       - install -Dm644 ../runtime/nvim.appdata.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml
       - sed -i "s|<id>nvim</id>|<id>$FLATPAK_ID</id>|" /app/share/metainfo/$FLATPAK_ID.metainfo.xml
-      # Install/fix icons
-      - mkdir -p /app/share/icons/hicolor/128x128
-      - install -Dm644 ../neovim-mark.svg /app/share/icons/hicolor/scalable/apps/nvim.svg
     sources:
       - type: archive
-        url: https://github.com/neovim/neovim/archive/v0.7.0.tar.gz
-        sha256: 792a9c55d5d5f4a5148d475847267df309d65fb20f05523f21c1319ea8a6c7df
+        url: https://github.com/neovim/neovim/archive/v0.8.0.tar.gz
+        sha256: 505e3dfb71e2f73495c737c034a416911c260c0ba9fd2092c6be296655be4d18
         x-checker-data:
           type: anitya
           project-id: 9037
           url-template: https://github.com/neovim/neovim/archive/v$version.tar.gz
       - type: patch
         path: nvim.desktop-exec-wrapper.patch
-      - type: archive
-        url: https://neovim.io/logos/neovim-logos.zip
-        sha256: c899d052fb8c31b124746fc33bde5a23ff8b02d323b42a0dd8dd66b57b81d20d
+
+  - name: nvim-icon
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 neovim-mark.svg /app/share/icons/hicolor/128x128/apps/nvim.svg
```

Comment:
svg is NOT 128x128

Reply:
it is not `128x128`, it is `scalable`

And yes icon are to be square, SVG or not.